marqeta/resources/collection.py

```diff
@@ -79,15 +79,15 @@ def list(self, endpoint=None, query_params=None, limit=None):
                 break
         return list_of_user_object
 
-    def create(self, data, endpoint=None, query_params=None):
+    def create(self, data, endpoint=None, query_params=None, proxy_data=None):
```

Comment:
Not sure where, but somewhere we should document what keys need to be passed as part of `proxy_data`.